packages/lab/stories/multiline-input/multiline-input.stories.tsx

```diff
@@ -0,0 +1,227 @@
+      <MultilineInput
+        readOnly
+        variant="secondary"
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput readOnly fullBorder defaultValue="Value" {...args} />
+      <MultilineInput
+        readOnly
+        fullBorder
+        variant="secondary"
+        defaultValue="Value"
+        {...args}
+      />
+    </FlowLayout>
+  );
+};
+
+export const EmptyReadonlyMarker: ComponentStory<typeof MultilineInput> = (
+  args
+) => {
+  return (
+    <FlowLayout>
+      <MultilineInput readOnly={true} {...args} />
+      <MultilineInput readOnly={true} emptyReadOnlyMarker="*" {...args} />
+    </FlowLayout>
+  );
+};
+
+export const Placeholder: ComponentStory<typeof MultilineInput> = (args) => {
+  return <MultilineInput placeholder={"Enter a value"} {...args} />;
+};
+
+export const TextAlign: ComponentStory<typeof MultilineInput> = (args) => {
+  return (
+    <FlowLayout style={{ maxWidth: "366px" }}>
+      <MultilineInput defaultValue={args.defaultValue ?? "Value"} {...args} />
+      <MultilineInput
+        textAlign="center"
+        defaultValue={args.defaultValue ?? "Value"}
+        {...args}
+      />
+      <MultilineInput
+        textAlign="right"
+        defaultValue={args.defaultValue ?? "Value"}
+        {...args}
+      />
+    </FlowLayout>
+  );
+};
+
+export const ValidationStates: ComponentStory<typeof MultilineInput> = (
+  args
+) => {
+  return (
+    <FlowLayout style={{ width: "366px" }}>
+      <MultilineInput validationStatus="error" defaultValue="Value" {...args} />
+      <MultilineInput
+        fullBorder
+        validationStatus="error"
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        validationStatus="warning"
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        fullBorder
+        validationStatus="warning"
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        validationStatus="success"
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        fullBorder
+        validationStatus="success"
+        defaultValue="Value"
+        {...args}
+      />
+    </FlowLayout>
+  );
+};
+
+export const WithAdornments: ComponentStory<typeof MultilineInput> = (args) => {
+  return (
+    <FlowLayout style={{ width: "366px" }}>
+      <MultilineInput
+        startAdornment={<Button variant="cta">AB</Button>}
+        endAdornment={
+          <>
+            <Text>%</Text>
+            <FilterClearIcon />
+          </>
+        }
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        startAdornment={<FlagIcon />}
+        endAdornment={
+          <Button>
+            <BookmarkSolidIcon />
+          </Button>
+        }
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        readOnly
+        startAdornment={<FlagIcon />}
```

Comment:
When using a button as a startAdornment in a readOnly state, the button is clickable. It is not consistent with the endAdornment.

Reply:
if it's readonly, it's the users responsibility to add disabled prop to the button as well (same in Input)
otherwise, we would have to add an input context for button, to ensure it is actually disabled (can't be done just through styling)